src/test/scala/iog/psg/cardano/util/ResourceFiles.scala

```diff
@@ -34,7 +36,10 @@ trait ResourceFiles { self: Assertions =>
 
   final def decodeJsonFile[T](file: String)(implicit dec: Decoder[T]): T = {
     val jsonStr = getJsonFromFile(file)
-    decode[T](jsonStr).getOrElse(fail(s"Could not decode $file"))
+    decode[T](jsonStr) match {
```

Comment:
This is unnecessary, right?

Reply:
well its useful because it adds error message to test failure